src/utils.cpp

```diff
@@ -82,7 +84,27 @@ bool TextChat::serialize(string *d)
     d->append((char*)&ou, sizeof ou);
     d->append((char*)&ts, sizeof(ts));
 
-    d->append("\0\0\0\0\0\0\0\0\0", 10); // additional bytes for backwards compatibility
+    char hasAttachments = attachedNodes.size();
```

Comment:
Correct. It would be very bad luck to overflow to zero, but it could happen. Fixed!